types/index.d.ts

```diff
@@ -47,8 +47,9 @@ declare namespace Handlebars {
   }
 
   export interface ParseOptions {
-      srcName?: string,
-      ignoreStandalone?: boolean
+      srcName?: string;
+      ignoreStandalone?: boolean;
+      disableWhitespaceControl?: boolean;
```

Comment:
Typings for the change are still missing here.

A test for the typings should go into `types/test.ts`. Please wrap the test into a new function like 

```ts
function testParseWithoutProcessing() {
   .... // call new function and make sure the the typings match
}
```

Reply:
There should also be mention of the function in the `docs/compiler-api.md` file.

Reply:
Awesome, thank you for the pointer on how to test the types. I've added a function that does this, mind reviewing to confirm I did it correctly?